src/domain/WeatherMood/components/Weather/WeatherRecommend.module.scss

```
@@ -108,14 +108,14 @@
     text-align: center;
     word-break: keep-all;
     background: f.color(white);
-    border: 1px solid f.color(orange-100);
+    border: 1px solid f.color(gray-300);
     border-radius: 12px;
-    @include m.text-style-extended(sm, 600, black);
+    @include m.text-style-extended(sm, 600, gray-700);
```

Stylelint (17.2.0)

[error] Expected empty line before at-rule (at-rule-empty-line-before)

(at-rule-empty-line-before)